packages/fxa-admin-panel/server/lib/client-config/index.ts

```diff
@@ -75,8 +77,12 @@ export class ClientConfig {
     const userGroupHeader = headers[USER_GROUP_HEADER];
     if (userGroupHeader != null) {
       clone.user.group = guard.getBestGroup(userGroupHeader);
+      logger.info('userGroupHeader', {
+        'userGroupHeader.notnull': userGroupHeader,
+      });
     }
-
+    logger.info('userGroupHeader', { 'userGroupHeader.null': userGroupHeader });
+    logger.info('headers', { headers });
```

Comment:
This is kinda weird, but these end up as cols in the logs db. Can we just do something like msg: JSON.stringify(headers)? Since this is just a temporary log probably better to just log out a message.

Reply:
Let's just keep these all these as logger.info('userGroupHeader', {...}) It'll make searching for the logs simpler.